packages/wallets/src/features/cashier/modules/Transactions/hooks/useTransactionTranslations.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+import { useTranslations } from '@deriv-com/translations';
+
+const useTransactionTranslations = () => {
```

Comment:
Maybe a better name? But, should this be a hook? It's just a mapper right?
```suggestion
const useTransactionLabels = () => {
```

Reply:
yes @nijil-deriv it's just a mapper, but we can't use `Localize` here because we are passing this translated labels into deriv-com/ui dropdown and the thing is that it only accepts regular string as an input initial value, thats why we need to use `localize()`  also we are reusing this hook across the Transactions module.

Reply:
@aum-deriv I believe you had similar concern regarding using `localize` which is marked as deprecated inside mappers, right? What was concluded after the discussion with @lubega-deriv ?

Reply:
@nijil-deriv @heorhi-deriv I've asked @ali-hosseini-deriv regarding this previously and he mentioned that we could just use the `localize` from deriv-com/translations for utils/mappers. It's currently marked as deprecated but if it's still in use we can remove the deprecated tag

Reply:
hi @nijil-deriv @lubega-deriv localize function is deprecated if we are using it directly from `import { localize } from '@deriv-com/translations'`, but if we are getting it from the `const { localize } = useTranslations()` hook we can use it across the app.

Reply:
@heorhi-deriv the deprecated tag for `localize` has been removed in [this PR](https://github.com/deriv-com/translations/pull/50) to avoid confusion as it's still needed for these scenarios where the `useTranslations` hook cannot be utilized. 

The package version can be updated to v1.3.5 and this method of utilizing `localize` in a mapper/util function could also be used alternatively